components/Globe.vue

```diff
@@ -83,17 +84,19 @@ const chartDefaultSettings: am5map.IMapChartSettings = {
 // Shared settings for countries.
 const countrySeriesSettings: am5map.IMapPolygonSeriesSettings = {
   geoJSON: WHONationalBorders,
-  fill: defaultLandColour,
 };
 // Settings for the 'background' world map, a map that includes all countries regardless
 // of whether they are supported in the model. These are the majority of the world's countries
 // and are not able to be selected.
 const backgroundSeriesSettings: am5map.IMapPolygonSeriesSettings = {
   ...countrySeriesSettings,
+  fill: unselectableLandColor,
+  stroke: defaultLandColour,
```

Comment:
so the faded stroke colour you're using here just happens to be the same as the selectable (default land colour)?
Wonder if you should make an alias const to be clear what's going on.  It's also a bit confusing because the boundaries in unselectable regions look a lot lighter than the selectable countries' fill - does the chart take the colour you give it and fade it anyway?

Reply:
yh annoyingly the chart auto fades it a bit, idea was to blend the boundaries in as much as possible to i wanted to use the colour the countries are filled in, ill make an alias i think!